tests/js/spec/redux/actions/substance.spec.jsx

```diff
@@ -0,0 +1,106 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+
+import {
+  SUBSTANCES_GET_REQUEST,
+  SUBSTANCES_GET_SUCCESS,
+  SUBSTANCES_GET_FAILURE,
+  SUBSTANCES_TOGGLE_SELECT_ALL,
+  SUBSTANCE_TOGGLE_SELECT,
+  substancesGetRequest,
+  substancesGetSuccess,
+  substancesGetFailure,
+  substancesGet,
+  substanceToggleSelect,
+  substancesToggleSelectAll,
+} from 'app/redux/actions/substance';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+describe('substance redux actions', function() {
+  beforeEach(function() {
+    moxios.install();
+  });
+
+  afterEach(function() {
+    moxios.uninstall();
+  });
+
+  const mockSubstance = {
```

Comment:
This test is a copy of the one in workbatches. I'll update this before removing WIP.